src/modm/platform/core/sam/linkerscript/sam_ram.ld.in

```diff
@@ -0,0 +1,47 @@
+%% import "../../cortex/linker.macros" as linker with context
+{{ linker.copyright() }}
+
+{{ linker.prefix() }}
+
+%% if vector_table_location == "ram"
+/* Round up the number of vector table entries to the nearest power-of-two and multiply by 8. */
+VECTOR_TABLE_ALIGNMENT = (1 << LOG2CEIL({{ number_of_interrupts + 16 }})) * 8;
+/* compute the vector table offset from start of RAM */
+VECTOR_TABLE_OFFSET = ALIGN(TOTAL_STACK_SIZE, VECTOR_TABLE_ALIGNMENT);
+%% else
+VECTOR_TABLE_OFFSET = TOTAL_STACK_SIZE;
+%% endif
+
+SECTIONS
+{
+{{ linker.section_vector_rom("FLASH") }}
+
+{{ linker.section_stack("RAM", "VECTOR_TABLE_OFFSET - TOTAL_STACK_SIZE") }}
+
+{{ linker.section_vector_ram("RAM") }}
+
+{{ linker.section_rom("FLASH") }}
+
+{{ linker.section_fastcode_in_flash() }}
+
+{{ linker.section_ram("RAM") }}
+
+{{ linker.section_heap("RAM", "heap1") }}
+
+{{ linkerscript_sections | indent(first=True) }}
+
+	/* TABLES! TABLES! ALL THE TABLES YOU COULD EVER WANT! TABLES! */
+{{ linker.section_table_zero(["bss"]) }}
+
+%% set copy_table = ["data"]
```

Comment:
I am wondering about the `.fastdata` section. Does it need explicit placing and/or copying?

Reply:
Yes, both. And it‘s also broken for stm32_ram.ld.in 😱. It only works because SystemClock::enable() writes these variables explicitly.

Reply:
> And it‘s also broken for stm32_ram.ld.in
Was still unsure about this. Do you or I fix it?

Reply:
I‘ll fix the STM32 linkerscript.